windows/window_manager_plugin.cpp

```diff
@@ -303,6 +303,12 @@ void WindowManagerPlugin::HandleMethodCall(
   } else if (method_name.compare("waitUntilReadyToShow") == 0) {
     window_manager->WaitUntilReadyToShow();
     result->Success(flutter::EncodableValue(true));
+  } else if (method_name.compare("forceRefresh") == 0) {

```

Comment:
These two methods don't seem necessary to be exposed on the dart side. Call it directly in the native method

Reply:
I've done so